commcare/src/main/resources/webapp/js/controllers.js

```diff
@@ -376,8 +376,7 @@
         $scope.saveAllowed = function() {
             return $scope.validateConfig()
                 && $scope.validateConfigName()
-                && $scope.validateUrlAndDomain()
-                && $scope.configOutdated;
```

Comment:
I removed it because when form was completed correctly and user go to another tab and returned to settings save button was disabled and it wasn't possible to save the config as configOutdated was set to false as it checks if configs properties have changed.